openstack/resource_openstack_compute_instance_v2.go

```diff
@@ -1484,7 +1495,7 @@ func getImageIDFromConfig(imageClient *gophercloud.ServiceClient, d *schema.Reso
 		return imageID, nil
 	}
 
-	return "", fmt.Errorf("Neither a boot device, image ID, or image name were able to be determined")
+	return "", fmt.Errorf("neither a boot device, image ID, or image name were able to be determined")
```

Comment:
See https://preply.com/en/blog/using-either-neither-nor-and-or-in-english/

```suggestion
	return "", fmt.Errorf("neither a boot device, image ID, nor image name could be determined")
```

Reply:
changes in error messages were just fixing IDE warnings :D